drivers/bus/pci/fdo.c

```diff
@@ -151,6 +151,12 @@ FdoEnumerateDevices(
                    PciConfig.VendorID,
                    PciConfig.DeviceID);
 
+            if (PciConfig.VendorID == 0 && PciConfig.DeviceID == 0)
```

Comment:
Out of curiosity, could you enable the DPRINTs that are above (line 147, 134, 121) and test that on VPC 2007 to see how the other values come out when this PCI device gets detected?

Reply:
Here's what I get with debugging enabled:
```
(drivers/bus/pci/fdo.c:124) Bus 0  Device  7  Func 2
(drivers/bus/pci/fdo.c:134) Size 64
(drivers/bus/pci/fdo.c:152) Bus 0  Device  7  Func 2  VenID 0x0000  DevID 0x0000
(drivers/bus/pci/fdo.c:156) Filter out devices with null vendor and device ID
```